docs/deploy_narrative.md

```diff
@@ -1,64 +1,109 @@
+4.  Create a single production image for appdev, next, & prod.
+5.  Tag the production image for deployment using the `deploy_tag.sh` script.
+6.  Deploy the production image in the relevant environments.
+
+### Deploying CI
+
+The Continuous Integration environment - <https://ci.kbase.us> - is intended for testing and demoing of new features that may not always work right. It's also intended to be continuously upgraded and redeployed, so this process is designed to be as easy as possible.
 
 #### Create CI Image
 
-1. Create a new temporary branch from the `develop` branch.
-1. Add a _single_ bug fix or new feature to your temporary branch.
-1. Create a new [pull request](https://github.com/kbase/narrative/compare) to merge your branch into `develop`.
-1. Ensure the automated [build test](https://github.com/kbase/narrative/actions?query=workflow%3A%22Build+Test+Image%22) completes successfully.
-1. Complete review and testing.
-1. Merge pull request.
+1.  Create a new temporary branch from the `develop` branch.
+2.  Add a _single_ bug fix or new feature to your temporary branch.
+3.  Create a new [pull request](https://github.com/kbase/narrative/compare) to merge your branch into `develop`.
+4.  Ensure the automated [build test](https://github.com/kbase/narrative/actions?query=workflow%3A%22Build+Test+Image%22) completes successfully.
+5.  Complete review and testing.
+6.  Merge pull request.
 
-#### Deploy CI
+#### Release CI
 
 Once a pull request is merged from a temporary branch to `develop` a new _development_ image will be available at:
 `docker.pkg.github.com/kbase/narrative/narrative-develop:latest`
 
 Simply restart the `narrative` container for CI in Rancher. No other action is needed.
 
-### Appdev, Next, & Prod
+### Stage a Final Release
+
+1.  Update versions following (roughly) [semantic versioning](https://semver.org). The files to be updated are:
+    -   `package.json`, `package-lock.json`, `bower.json`, `src/config.json`, `src/config.json.templ`, `src/biokbase/narrative/\_\_init\_\_.py`
+    -   Major - backward incompatible changes, like the move from Python 2 -> Python 3, or Narrative typed object changes. These are generally considered to be changes that have a strong impact on the Narrative Interface and are hard or impossible to move backward from.
+    -   Minor - new features that don't affect compatibility.
+    -   Patch - adjustments to existing features, bug fixes.  
+
+2.  Add release notes in `RELEASE_NOTES.md`
+    -   Add a new heading with the updated version.
+    -   Add a list of changes with some reasonable detail. You don't need to add the name of every module changed, just the overall feature affected. Include JIRA ticket numbers where appropriate.
+
+3.  PR these changes to the develop branch.
+    -   Ensure that tests pass.
 
-#### Create Image
+4.  PR the develop branch to the master branch.
+    -   Deploy on the next environment.
```

Comment:
How do you deploy on the next environment?

Reply:
Same comment as above - next, appdev, and prod need an operator to do the deployment. I'll add a note about that.

Reply:
Added a note.